### PR TITLE
created normalization unit test

### PR DESCRIPTION
**Pull request by tomsilver**
addressing #126 

@BoltzmannBrain please review

### CODE REVIEW

File: tests/unit/normalize_test.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Tests normalization scheme for normalizing detectors' scores according
+to baseline."""
+
+import csv
+import json
```
Comment (BoltzmannBrain) [inline tests/unit/normalize_test.py]: use simplejson, as imported [here](https://github.com/numenta/NAB/blob/master/nab/labeler.py#L27-30)

File: tests/unit/normalize_test.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+import os
+import shutil
+import tempfile
+import unittest
+
+from nab.runner import Runner
+
+
+def createTemporaryCsv(parentDir, fileName, data):
```
Comment (BoltzmannBrain) [inline tests/unit/normalize_test.py]: change name b/c there's nothing about this method that requires the csv (or its dir) to be temporary

File: tests/unit/normalize_test.py [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    testRunner.resultsFiles = [resultsFilePath]
+
+  return testRunner
+
+
+
+class NormalizationTest(unittest.TestCase):
+
+  def setUp(self):
+    self._tmpDirs = [] # stores absolute paths of dirs created during testing
```
Comment (BoltzmannBrain) [inline tests/unit/normalize_test.py]: `setUp()` and `tearDown()` are called for each test method, so this is unnecessary. If a future test creates multiple temp dirs we'll modify these methods then.

File: tests/unit/normalize_test.py [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    self.resultsHeaders = ['Detector','Profile','Score']
+
+
+  def tearDown(self):
+    for tmpDir in self._tmpDirs:
+      shutil.rmtree(tmpDir)
+
+
+  def _createTemporaryResultsDir(self):
```
Comment (BoltzmannBrain) [inline tests/unit/normalize_test.py]: don't need this (see above comment)

**Comment by BoltzmannBrain** [comment]
@tomsilver done with review, left some comments, otherwise :+1:

**Comment by tomsilver** [comment]
@BoltzmannBrain thanks, ready for re-review!

**Comment by BoltzmannBrain** [comment]
:+1:

**Pull request merged**